config/hooks/normal/0520-Qtpyvcp-repos.hook.chroot

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+curl sS https://repository.qtpyvcp.com/repo/kcjengr.key | gpg --dearmor | sudo tee /usr/share/keyrings/kcjengr.gpg
+sudo gpg --homedir /tmp --no-default-keyring --keyring /usr/share/keyrings/QTPYVCP.gpg --keyserver keys.openpgp.org --recv-keys 2DEC041F290DF85A
+sudo echo "deb [arch=amd64 signed-by=/usr/share/keyrings/QTPYVCP.gpg] https://repository.qtpyvcp.com/apt develop main " | sudo tee -a /etc/apt/sources.list.d/qtpyvcp.list >/dev/null
```

Comment:
The 'sudo' in front of the echo command serve no purpose.

Reply:
Again I used the QTPYVCP supplied format. But yes we're are root so can be cleaned up.

Reply:
Moved to /etc/apt/keyrings as suggested